client/components/training-session.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import moment from 'moment';
+
+export default class TrainingSession extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      trainingSession: [],
+      startDate: this.props.date
+    };
+  }
+
+  async componentDidMount() {
+    const date = moment(this.state.startDate).format();
+    const sessionDate = date.slice(0, 10);
```

Comment:
I don't feel great about this date formatting and slicing thing. Too much hidden intent.

I _think_ you are trying to "floor" the date?

Moment has a way for you to calculate that
https://momentjs.com/docs/#/manipulating/start-of/

And you can then format the string exactly as you want
https://momentjs.com/docs/#/displaying/format/

Using those will help the reader of your code a lot.